src/main/java/projet/projetstage02/utils/PDFCreationUtil.java

```diff
@@ -0,0 +1,95 @@
+package projet.projetstage02.utils;
+
+import com.itextpdf.text.*;
+import com.itextpdf.text.pdf.PdfWriter;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.util.Base64;
+import java.util.Map;
+
+public class PDFCreationUtil {
+    /***
```

Comment:
Why is this there?

Reply:
In case you or someone else needs to use it. it's a bit counter intuitive so i figured adding documentation wouldn't hurt